src/Covid19Dashboard/ChartCarousel/ChartCarousel.less

```diff
@@ -42,3 +42,44 @@
   white-space: pre-wrap;
   z-index: 10; // place it over the "mapbox" link...
 }
+
+/** override some style for carousel's arrow and dots **/
```

Comment:
should we deploy this portal branch to qa-covid19 to test that the carousels are fine?

Reply:
Good idea. I have tested locally against PRC and i think the carousels looks fine to me.

But I agree to deploy this to qa-covid19 so we can have more eyes on this to make sure I didn't miss anything

Reply:
https://github.com/uc-cdis/gitops-qa/pull/986 @paulineribeyre